amical/mf_pipeline/bispect.py

```diff
@@ -989,13 +990,16 @@ def _add_infos_header(infos, hdr, mf, pa, filename, maskname, npix, verbose=True
 
     astropy_version = astropy.__version__
     working_version = "5.0rc1"
-    if Version(astropy_version) < Version(working_version):
+    hdr_commentary_keys = fits.Card._commentary_keywords
+    if any(hck in hdr for hck in hdr_commentary_keys) and (
+        Version(astropy_version) < Version(working_version)
```

Comment:
same as my previous comment; I'd suggest storing the runtime version in a const (`ASTROPY_VERSION = Version(astropy.__version__)`) but not the requirement, which should be parsed only if needed, e.g. directly in the `if` statement

Reply:
Fixed by the last commit. 5.0rc is used directly in the `if` statement now.